.setup/install_system.sh

```diff
@@ -666,7 +686,7 @@ if [ ${WORKER} == 0 ]; then
         echo "Creating submitty master database"
         su postgres -c "psql -c \"CREATE DATABASE submitty WITH OWNER ${DB_USER}\""
         su postgres -c "psql submitty -c \"ALTER SCHEMA public OWNER TO ${DB_USER}\""
-        python3 ${SUBMITTY_REPOSITORY}/migration/run_migrator.py -e master -e system migrate --initial
+        python3 ${SUBMITTY_REPOSITORY}/migration/run_migrator.py -c ${SUBMITTY_INSTALL_DIR}/config -e master -e system migrate --initial
```

Comment:
Is this change necessary? the `run_migrator.py` script should be automatically picking up the config directory, and I'm a bit worried on why it wouldn't work under arm.

Reply:
I don't think this was necessary, I was incrementally debugging and I think I was running something from the wrong directory so the default didn't work.  It shouldn't hurt, right?  It might make future installation debugging easier?